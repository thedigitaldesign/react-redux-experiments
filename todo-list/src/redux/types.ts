// Todo List
export const TodoListTypes = {
  TODO_LIST: 'TODO_LIST'
}

export const TodoItemTypes = {
  TODO_ITEM: 'TODO_ITEM'
}

export const UsernameTypes = {
  USERNAME: 'USERNAME'
}
