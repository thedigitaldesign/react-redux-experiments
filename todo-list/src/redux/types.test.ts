import * as enz from 'enzyme'

// Dispatcher Types
import * as type from './types'

describe('Dispatch Types List', () => {
  it('should contain dispatch types for `TODO_ITEM`', () => {
    expect(type.TodoItemTypes).toMatchSnapshot()
  })

  it('should contain dispatch types for `TODO_LIST`', () => {
    expect(type.TodoListTypes).toMatchSnapshot()
  })

  it('should contain dispatch types for `USERNAME`', () => {
    expect(type.UsernameTypes).toMatchSnapshot()
  })
})
