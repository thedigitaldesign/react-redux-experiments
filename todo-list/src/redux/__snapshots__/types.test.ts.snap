// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dispatch Types List should contain dispatch types for \`Todo\` 1`] = `undefined`;
